docs/synapse/userguides/data_model.rstorm

```diff
@@ -132,33 +181,37 @@ A **node** is a unique object within Synapse; they are specific instances of gen
 
 - A **primary property**, represented by the form of the node plus its value (``<form> = <valu>``). All primary
   properties must be unique for a given form; the uniqueness of the ``<form> = <valu>`` pair ensures there can 
-  be only one node in Synapse that represents the domain ``woot.com``.
-  
-  Because the unique pair "defines" the node, the comma-separated form / value combination (``<form>,<valu>``)
-  is also known as the node’s :ref:`gloss-ndef` (short for "node definition").
+  be only one node in Synapse that represents the domain woot.com ( ``inet:fqdn = woot.com`` ). Because this
+  unique pair "defines" the node, the comma-separated form / value combination (``<form>,<valu>``) is also known
+  as the node’s :ref:`gloss-ndef` (short for "node definition").
 
-- One or more **universal properties.** As the name implies, universal properties are applicable to all nodes.
+- One or more **universal properties** and an associated property value. As the name implies, universal properties
+  apply to all nodes.
 
-- Optional **secondary properties.** Similar to primary properties, secondary properties consist of a property name
+- Optional **secondary properties** Similar to primary properties, secondary properties consist of a property name
```

Comment:
```suggestion
- Optional **secondary properties**. Similar to primary properties, secondary properties consist of a property name
```

Reply:
Wasn't sure if this should still have the period here since there is a capital S "Similar", or if this was intended to not need the period.